x/evm/keeper/state_transition.go

```diff
@@ -325,13 +294,15 @@ func (k *Keeper) RefundGas(msg core.Message, leftoverGas uint64) error {
 		// no refund, consume gas and update the tx gas meter
 	}
 
+	return nil
+}
+
+func (k *Keeper) setGasMeterAndConsumeGas(msg core.Message, gasUsed uint64) {
 	// set the gas consumed into the context with the new gas meter. This gas meter will have the
 	// original gas limit defined in the msg and will consume the gas now that the amount has been
 	// refunded
```

Comment:
can you move this above the function so that it is used as the godoc comment?

Reply:
moved